tools/nanoplot/nanoplot.xml

```diff
@@ -0,0 +1,498 @@
+<tool id="nanoplot" name="NanoPlot" version="1.13.0">
+    <description>Plotting suite for Oxford Nanopore sequencing data and alignments</description>
+    <requirements>
+        <requirement type="package" version="1.13.0">nanoplot</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+        <![CDATA[
+        #if $reads.type == 'pickle'
+            ln -s '$reads.files' './read.$reads.files.ext' &&
+        #else
+
+            reads='' &&
+            #for $i, $f in enumerate($reads.files)
+                #if $f
+                    #set $extension = $f.ext
+                    #if $extension == 'fastqsanger'
+                        #set $extension = 'fastq'
+                    #end if
+                    ln -s '$f' './read_${i}.$extension' &&
+                    reads="\$reads read_${i}.$extension" &&
+                #end if
+            #end for
+            reads=\$(echo \$reads | xargs) &&
+        #end if
+
+        NanoPlot
+            #if $reads.type == 'pickle'
+                --pickle './read.$reads.files.ext'
+            #else
+                --$reads.type \$reads
+            #end if
+
+            #if $filter.maxlength
+                --maxlength $filter.maxlength
+            #end if
+            #if $filter.minlength
+                --minlength $filter.minlength
+            #end if
+            #if $filter.downsample
+                --downsample $filter.downsample
+            #end if
+            #if $filter.minqual
+                --minqual $filter.minqual
+            #end if
+            #if $filter.readtype
+                --readtype $filter.readtype
+            #end if
+            #if $customization.color
+                --color $customization.color
+            #end if
+            #if $customization.format
+                --format $customization.format
+            #end if
+            #if $customization.plots
+                --plots ${str($customization.plots).replace(',', ' ')}
+            #end if
+            $filter.drop_outliers
+            $filter.loglength
+            $filter.percentqual
+            $filter.alength
+            $filter.barcoded
+            $customization.noN50
+            $customization.N50
+            #if $customization.title
+                --title '$customization.title'
+            #end if
+            $store
+            $raw
+            -o outputs &&
+            mv outputs/*.log outputs/NanoPlot_log.txt &&
```

Comment:
If we print the log file to stdout we do not need to put them as real output. I do not consider them as useful enough to store them as datasets.

So a simple `cat outputs/*.log` might be enough and will simplify our outputs of this tool dramatically.